popup.js

```diff
@@ -126,6 +128,21 @@ function addRowBehavior(div) {
   };
 }
 
+// Returns a debounced version of the provided function
```

Comment:
Nit, can you use /* style for this comment like the other function level comments?